dart-sdk/rust/src/runtime.rs

```diff
@@ -0,0 +1,36 @@
+pub(crate) use inner::*;
+
+#[cfg(not(target_arch = "wasm32"))]
+mod inner {
+    pub(crate) type FlutterRustBridgeRuntime = tokio::runtime::Handle;
+
+    pub(crate) fn get_runtime() -> FlutterRustBridgeRuntime {
+        crate::frb_generated::FLUTTER_RUST_BRIDGE_HANDLER
+            .async_runtime()
+            .0
+             .0
```

Comment:
If the underlying interfaces changes do you think the compiler will break here and alert us? Consider how to write this with protection and a log message to help a future developer.

Reply:
Yes, absolutely. That's one of the things that broke in web build because implementation is different.

We are pinning `rust_flutter_bridge` version, so this should not break unexpectedly. And when it does, the error will be scoped to this file, so it will be quite clear what happened